project.clj

```diff
@@ -51,6 +51,7 @@
                        {:id "advanced"
                         :source-paths ["src/test/clojurescript" "src/test/common"]
                         :compiler {:output-to "target/js/advanced.js"
+                                   :anon-fn-naming-policy :mapped
```

Comment:
What’s this for?

Reply:
It seems that `:advanced` optimizations by default removes the anonymous function names, i had to provide this new property such that the fn names would remain to properly test the changes.

Reply:
Ah ok. I overlooked some of the fn to name stuff happening in like tests and inspect. I think during inspection the cljs would still have an issue right? 

I’m a little uncertain of the parts of these changes than involve pulling fn names from their fn impl. But I suppose if it seems to work for now. The clj side I’d consider a bit more reliable than the cljs.